docs/API.md

```diff
@@ -57,14 +54,30 @@ function myApplicationLogic() {
 }
 ```
 
-##### Scope
+### Use the Scope manager with the SignalFx Tracing Library for JavaScript
+
+To provide span context propagation within a Node.js application, the library
+includes a scope manager. A scope manager is a utility for registering and
+providing a span that can cross both synchronous and asynchronous contexts.
+The span it provides is registered as active, and you can use it for noting an
+accessor's execution state and for parenting child spans. This means you can
+reference an existing span in a particular section of traced functionality
+without already explicitly passing the span as an argument.
+
+The scope manager is available via `tracer.scope()`. Use its return value in a
+global context. It has three methods for active span management:
+
+* scope.active()
+* scope.activate(span, fn)
+* scope.bind(target, [span])
+
+Because the scope manager isn't defined in the current OpenTracing JavaScript
```

Comment:
Repeated information

Reply:
i think we removed the first instance of this info above.